logging/logging.go

```diff
@@ -39,33 +40,33 @@ func SetGlobalLoggerLevel(l Level) {
 	globalLogLevel = l
 }
 
-func GetDefaultLogger() Logger {
-	return defaultLogger
+func GetGlobalLogger() Logger {
```

Comment:
Actually we may export functions like `Infof`, `Warn` as delegates to the default logger in `logging` package, so it could be more convenient to use:

```go
logging.Infof(xxx)
```